### PR TITLE
Ezamoraa/qr secret solver

### PR DESCRIPTION
**Pull request by ezamoraa**

### CODE REVIEW

File: final_project/src/final_project/qr_secret_solver.py [hunk 1]
```diff
@@ -0,0 +1,148 @@
+#!/usr/bin/env python3
+
+import rospy
+import actionlib
+import math
+from move_base_msgs.msg import MoveBaseAction, MoveBaseGoal
+from std_msgs.msg import Int8, String
```
Comment (Tran97) [inline final_project/src/final_project/qr_secret_solver.py]: We are not using Int8 anywhere

**Review by Tran97 — CHANGES_REQUESTED** [review]
looks good overall

File: final_project/src/final_project/qr_secret_solver.py (hunk 1 above)
Comment (ezamoraa) [reply final_project/src/final_project/qr_secret_solver.py]: fixed

**Review by Tran97 — APPROVED** [review]
Muy Bien

**Pull request merged**